### ISSUE TITLE
About data

### ISSUE THREAD

**Issue by Cherry1210** [OrangeUW/AutoML4ETC#1]
How can the original pcap be turned into the .mat form needed in the code? Thanks.

**Comment by navidpadid** [comment]
Thank you for your comment.

1. ".mat" file format is not necessary for the AutoML4ETC, the training objects are standard Tensorflow/Keras training format:
`# Train_dataset and test_dataset inputs are a generator or keras.utils.Sequence and classes is the number of classes in the output
automl.search(train_dataset=train, test_dataset=test, input_shape=(1024, 3), classes=5) `

2. The pcap to .mat form is not part of the AutoML4ETC framework and is the dataset pre-processor. We may add the pre-processor code in the future as well, but it's not in the scope of the AutoML4ETC tool.

**Comment by 529669517** [comment]
I have the same question, how's the training data looks like?

**Comment by 529669517** [comment]
Did you normalize the bytes into range [0,1] before feeding it into neural network?

**Comment by navidpadid** [comment]
no, but there could be batch normalization layers in between the nodes.
